book/graphics.md

```diff
@@ -7,8 +7,8 @@ next: text
 
 A web browser doesn't just download a web page; it also has to show
 that page to the user. In the twenty-first century, that means a graphical
-application. So in this chapter we'll equip our browser with a
-graphical user interface.[^1]
+application.[^1] So in this chapter we'll equip our browser with a
+graphical user interface.
 
 [^1]: There are some obscure text-based browsers: I used `w3m` as my
```

Comment:
```suggestion
[^text-browsers]: There are some obscure text-based browsers: I used `w3m` as my
```

Reply:
Fun fact: all of the numeric footnotes date to my original conversion of the book, in winter 2020, from org-mode to markdown. Amazing that there are still some numeric footnotes in there!